package.json

```diff
@@ -8,7 +8,9 @@
     "start": "next start",
     "lint": "next lint",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "jest",
+    "test:watch": "jest --watchAll"
```

Comment:
you haven't written any tests yet, normally, I would only install a package once I am actually going to use it.
In your specific case, once you write the first tests.

If you write tests in your next user story, it is fine to leave the package as it is, otherwise, I would remove it from the `package.json` and run `npm install` afterwards

Reply:
Thank you, Niklas! I removed comment and dependencies and ran npm install. I also added a tiny ternary to change the text within the rekordbag to change depending on whether it is empty or not. 
```
 {numberSelected === 0 ? (
        <Text>
          Your bag is empty. <br />
          Go back digging in you collection.
        </Text>
      ) : (
        <Text>
          You have selected <strong>{numberSelected}</strong> records:
        </Text>
      )}
```